cmd/runtimecfg/node-ip.go

```diff
@@ -113,10 +114,11 @@ func set(cmd *cobra.Command, args []string) error {
 		nodeIPs += "," + chosenAddresses[1].String()
 	}
 	remoteWorker := false
-	// if chosen ip doesn't match vips need create a file that
-	// this file will be used by keepalived container to verify if it should run or not
-	// We want to disable keepalive in case this host is remote worker
-	if len(vips) > 0 && !matchesVips {
+	// if chosen ip doesn't match vips, we need create a file that
+	// will be used by keepalived container to verify if it should run or not
+	// We want to disable keepalived in case this host is remote worker
+	// This is skipped in case of user managed load balancer as the VIPs are not managed by keepalived.
+	if len(vips) > 0 && !matchesVips && !userManagedLB {
```

Comment:
Does it mean that remote worker node becomes incompatible with user-managed LB? Are we fine with it?

Reply:
That doesn't make me happy but we haven't come up with anything better so far.
@tsorya @cybertron could you think of another way to detect if a node is a remote-worker?

Reply:
I'll be nice if we wouldn't block remote workers to work with an external LB.

Reply:
This doesn't block remote workers with external LB, does it? Keepalived won't be deployed on those nodes in either case. If you're using external LB then we don't need the remote worker logic because everything is essentially remote.

Reply:
Yep, agree with @cybertron. With external LB all the workers are actually remote one so it doesn't really matters

Reply:
I'm talking about the labelling, which won't happen if remote-worker + external-LB (in this code).

Reply:
we discussed this offline and I think this is a viable option now. Since we don't need labelling when an external LB is in play anyway.